aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -37,81 +26,159 @@ use constant ANACONDA_VERSION_EL_6_0 => version->new("13.21");
 use constant ANACONDA_VERSION_EL_7_0 => version->new("19.31");
 use constant ANACONDA_VERSION_LOWEST => ANACONDA_VERSION_EL_5_0;
 
-use parent qw (NCM::Component);
+# Configuration option names
+use constant RESCUEBOOT => 'rescueconfig';
+
+# Import PXE-related constants shared with other modules
+use NCM::Component::PXELINUX::constants qw(:all);
+
+# Support PXE variants and their parameters (currently PXELINUX and Grub2)
+# 'name' is a descriptive name for information/debugging messages
+# To add a new variant, define its name in PXELINUX::constants module, create a
+# new xxx_VARIANT_PARAMS with the same structure as the existing one and add it
+# in @VARIANT_PARAMS list. Then update the code where appropriate.
+Readonly my %GRUB2_VARIANT_PARAMS => (name => 'Grub2',
+                                      nbpdir_opt => NBPDIR_GRUB2,
+                                      kernel_root_path => GRUB2_EFI_KERNEL_ROOT,
+                                      format_method => 'write_grub2_config');
```

Comment:
should be with `_` prefix
but there should also be a unittest that fails with the current value, try to make one first